### DISCUSSION TITLE
I want to disable the behavior where info lines in the top right corner change position if the player has a status effect.

### DISCUSSION THREAD

**Discussion by HydraheadHunter** [maruohon/minihud#236] [Q&A] [answered]
I keep my info lines on the right side of the screen, and when the player has an status effect, this causes the lines to move down the screen so as to not obscure the status effects. This is a good feature, but, I have a second mod that shows the durability of my items  which is positioned below the infolines I use, and when the player has a status effect, the info lines and the item durabilities overlap.

My solution for this was to put the status effects on the unused left side of the screen using a third mod, and while this eliminated the need for the behavior, it understandably didnot change the behavior. What file would I need to edit in my fork to disable the behavior where if the player has a status effect and the infolines are in the top right corner, draw the info lines lower down on the screen?

**Comment by maruohon** [comment, marked as answer]
Looks like it currently happens "hard-coded" in malilib's `RenderUtils.renderText()` method. So to disable that, you just need to comment out this line: https://github.com/maruohon/malilib/blob/adddaa833e0ea9a6323406c6704014e47ec62419/src/main/java/fi/dy/masa/malilib/render/RenderUtils.java#L404

I guess I'll change around some things in the main development code so that there will be a config option for this in future versions.

**Reply by HydraheadHunter** [reply]
Thanks a lot. I appreciate the help.
